src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -68,6 +68,9 @@ public partial class DynamoView : Window, IDisposable
         private ShortcutToolbar shortcutBar;
         private bool loaded = false;
 
+        // list of items in the right side bar. 
+        private List<TabItem> _tabItems = new List<TabItem>();  
```

Comment:
For fields in c# I don't believe we follow this style - 
https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/field